cloudinary-core/src/main/java/com/cloudinary/metadata/Restrictions.java

```diff
@@ -0,0 +1,36 @@
+package com.cloudinary.metadata;
+
+import java.util.HashMap;
+
+/**
+ * Represents the restrictions metadata field.
+ */
+public class Restrictions {
+
+    HashMap restrictions = new HashMap();
```

Comment:
maybe make it private and create a method toHash() or something. I would check other classes, to make it consistent